client/src/components/Questions/QuestionsAskBar/QuestionsAskBar.spec.js

```diff
@@ -23,4 +23,4 @@ describe('QuestionsAskBar unit tests', () => {
     wrapper.find('button').simulate('click')
     expect(mockFn).toHaveBeenCalled();
```

Comment:
wow nice!! cool test

Reply:
that is an old one. it just never appeared on github until now :)